src/SALib/analyze/morris.py

```diff
@@ -127,15 +135,16 @@ def analyze(
     num_trajectories = int(Y.size / (number_of_groups + 1))
     trajectory_size = int(Y.size / num_trajectories)
 
-    elementary_effects = _compute_elementary_effects(X, Y, trajectory_size, delta)
+    elementary_effects, see_trajectory = _compute_elementary_effects(X, SX, Y, trajectory_size, delta)
```

Comment:
See my larger note. Consider changing this to two call to the `_compute_elementary_effects()` function:

```python
delta = _compute_delta(num_levels)
elementary_effects = _compute_elementary_effects(X, Y, trajectory_size, delta)
scaling_delta = _calculate_step_size_x(SX)
see_trajectory = _compute_elementary_effects(SX, Y, trajectory_size, scaling_delta)
```